mindsdb/integrations/handlers/intercom_handler/README.md

```diff
@@ -0,0 +1,73 @@
+## Intercom API Handler Initialization
+
+The Intercom API handler can be initialized with the following parameters:
+
+- `access_token` - your Intercom access token
+
+## Implemented Features
+
+- `Articles` - retrieve articles from Intercom
```

Comment:
Maybe we can add TODO with the new features

Reply:
Done